roles/haproxy/tasks/main.yml

```diff
@@ -3,8 +3,11 @@
   template: src=etc/default/haproxy dest=/etc/default/haproxy
   notify: reload haproxy
 
+- name: haproxy ppa key
+  apt_key: id=505d97a41c61b9cd keyserver=keys.gnupg.net state=present
```

Comment:
Will the `keyserver=` still try and call out to that server? How does it handle if it cannot be reached?